sdk/eventhub/event-hubs/src/diagnostics/tracing.ts

```diff
@@ -5,6 +5,11 @@ import { createTracingClient, TracingSpanOptions, TracingSpanKind } from "@azure
 import { EventHubConnectionConfig } from "../eventhubConnectionConfig";
 import { packageJsonInfo } from "../util/constants";
 
+/**
+ * The names of the operations that can be instrumented.
+ */
+export type MessagingOperationNames = "publish" | "receive" | "process" | "settle";
```

Comment:
I don't think we have this settlement concept in EH, @jeremymeng is that right?

Reply:
Removed "settle" for now

Reply:
we use "settle" for checkpointing in .NET and Java.